### PR TITLE
BISDN Linux now supports Edgecore AS5835-54X

### PR DESCRIPTION
**Pull request by ideaship**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ The BISDN Linux Distribution is available for the following whitebox switch plat
 * Delta AG7648
 * Edgecore AS4610 series
 * Edgecore EPS202 (AS4630-54PE)
+* Edgecore AS5835-54X
```
Comment (KanjiMonster) [inline index.md:28]:
```suggestion
* Edgecore DCS201 (AS5835-54X)
```

Comment (ideaship) [reply index.md:28]: Can we either add the new names for all switches or always have the AS name first? Switching columns for the same kind of name is annoying readers.

Comment (KanjiMonster) [reply index.md:28]: Feel free to create a PR for that (maybe after this one was merged, so it doesn't become a blocker)

Comment (ideaship) [reply index.md]: Applied.

**Review by KanjiMonster — APPROVED** [review]

**Pull request merged**